packages/accordion-react/example/index.html

```diff
@@ -0,0 +1,7 @@
+<html>
+    <link rel="stylesheet" href="../../accordion/accordion.scss" />
```

Comment:
Kan ikke gå uten for rot i pakken sin. Om man trenger noe utenfor sin egen pakke, så må man importere som om det var en pakke på npm.